src/core/domain-services/crux-messenger.ts

```diff
@@ -35,11 +36,21 @@ export class CertificateManager {
 
 export class EncryptionManager {
     // Use ECIES to encrypt & decrypt
-    public static encrypt = (content: string, pubKeyOfRecipient: string): string => {
-        return content;
+    public static encrypt = async (content: string, pubKeyOfRecipient: string): Promise<string> => {
+        const toEncrypt = Buffer.from(content, "utf8");
```

Comment:
Please add a TODO here -this UTF8 is going to bother us in the future 
"TODO - Handle encoding properly"

Reply:
Done

Reply:
You haven't addressed this either

Reply:
Added